content/en/tracing/trace_collection/open_standards/otel_collector_datadog_exporter.md

```diff
@@ -10,86 +10,91 @@ further_reading:
+1. Download the latest release of the OpenTelemetry Collector Contrib distribution, from [the project's repository][3].
 
-The exporter attempts to get a hostname by checking the following sources in order, falling back to the next one if the current one is unavailable or invalid:
+2. Create a configuration file and name it `collector.yaml`. Please follow the [Configuring the Datadog Exporter](#configuring-the-datadog-exporter) section below.
 
-1. Hostname set in the OTLP resource
-1. Manually set hostname in the exporter configuration
-1. Cloud provider API hostname
-1. Kubernetes hostname
-1. Fully qualified domain name
-1. Operating system host name
+3. Run the collector, specifying the configuration file using the `--config` parameter:
 
-## Ingesting OpenTelemetry traces with the collector
-
-<div class="alert alert-warning">
-The Datadog exporter for the OpenTelemetry Collector is currently in beta. It may consume high CPU and memory resources. Configuring particularly the pipeline and batch processor may take some iteration before it responds with accurate metrics given your production environment. <a href="https://docs.datadoghq.com/help/">Reach out to support</a> if it doesn't work as you expect.
-</div>
+      ```
+      otelcontribcol_linux_amd64 --config collector.yaml
+      ```
 
-The OpenTelemetry Collector is configured by adding a [pipeline][7] to your `otel-collector-configuration.yml` file. Supply the relative path to this configuration file when you start the collector by passing it in via the `--config=<path/to/configuration_file>` command line argument. For examples of supplying a configuration file, see the [environment specific setup](#environment-specific-setup) section below or the [OpenTelemetry Collector documentation][8].
+## Configuring the Datadog Exporter
 
-Here is an example trace pipeline configured with an `otlp` receiver, `batch` processor and `datadog` exporter:
+To use the Datadog Exporter, simply add it to your [OpenTelemetry Collector configuration][4]. Here is a bare-bones configuration file that is ready-to-use as soon as you have set your Datadog API key as the DD_API_KEY environment variable:
 
-```
+```yaml
 receivers:
   otlp:
     protocols:
-      grpc:
       http:
+      grpc:
 
 processors:
   batch:
 
 exporters:
   datadog:
-    host_metadata:
-      tags:
-        - example:tag
     api:
-      key: aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
-      site: {{< region-param key="dd_site" code="true" >}}
+      key: ${DD_API_KEY}
 
 service:
   pipelines:
+    metrics:
+      receivers: [otlp]
+      processors: [batch]
+      exporters: [datadog]
     traces:
       receivers: [otlp]
       processors: [batch]
       exporters: [datadog]
 ```
 
-## Environment specific setup
+The above configuration enables the receiving of OTLP data from OpenTelemetry instrumentation libraries via HTTP and gRPC and sets up a [batch processor][5] which is mandatory for any non-development environment.
```

Comment:
> which is mandatory for any non-development environment
Can we maybe add a few words to clarify why it's mandatory? E.g. do we require it or does OpenTelemetry? What happens if they don't use it in production? Etc.

Don't need to go into depth, but the word mandatory raises questions that we can ideally answer with some simple text.